_schemaV2.graphql

```diff
@@ -12574,7 +12574,11 @@ type Sale implements Node {
   extendedBiddingPeriodMinutes: Int
 
   # A formatted description of when the auction starts or ends or if it has ended
-  formattedStartDateTime: String
+  formattedStartDateTime(
+    # Formatted date that shows if a date range if a sale is open, Closed {date}
+    # if the sale is closed, and Lots are closing if the sale is ending
+    summary: Boolean = false
+  ): String
```

Comment:
Since formattedStartDateTime is used all over, wouldn't this be a large backward-incompatable breaking change?